TA_dataset/bin/dataset_api.py

```diff
@@ -102,17 +102,20 @@ def ds_lrq_facet_values(
             filter_=filter, name=name, max_values=max_values
         ),
     )
-    return ds_lrq_run_loop(client=client, body=body)
+    body.additional_properties = get_tenant_related_payload(acc_conf)
+    return ds_lrq_run_loop(logger, client=client, body=body)
 
 
 # Executes LRQ run loop of launch-ping-remove API requests until the query completes
 # with a result
 # Returns tuple - value, error message
 def ds_lrq_run_loop(
-    client: AuthenticatedClient, body: PostQueriesLaunchQueryRequestBody
+    log,
```

Comment:
Do we really need to pass `log` here? Just a few lines below there is a `logger()` call

Reply:
logger() wont create the logs in the index="_internal" so using the standard logger instead of logger()

Reply:
Could you add this to the doc string comment where neeeded, please? So the next person who will be looking on this code is aware of this.